### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ke-Jin-Laserfiche**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,5 +12,37 @@ Contains resource files that can be used in conjunction with LfLocalizationServi
...
+```ts
+let localizationService = new LfLocalizationService();
+localizationService.setLanguage('es');
+await localizationService.initResourcesFromUrlAsync('https://cdn.jsdelivr.net/npm/@laserfiche/lf-resource-library@1.0.0/resources/laserfiche-base');
+let loading =  localizationService.getString('LOADING'); // loading -> 'Cargando...'
+```
+
+# [Icons](https://github.com/Laserfiche/lf-resource-library/blob/main/resources/icons/document-icons-documentation.md)
+
+[**Example Usages**](https://github.com/Laserfiche/lf-js-utils/blob/7ce0108/src/utils/icon-utils.ts#L10)
```
Comment (alexgomezlf) [inline README.md:45]: does this point to icon-utils file?

**Review by alexgomezlf — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (Ke-Jin-Laserfiche) [reply README.md]: it should only point to the function that uses the icons file. fixed

File: README.md [hunk 2]
```diff
@@ -12,5 +12,42 @@ Contains resource files that can be used in conjunction with LfLocalizationServi
    - If a resource key is not found in the selected language, the en string will be used.
 5. Optional Language: For Laserfiche, we recommend providing resources for at least: ar, en-US, en-GB, es, fr-CA, fr, pt-BR, zh-Hans, zh-Hant
 
+# Documentation
+
+## With LfLocalizationService from [`@laserfiche/lf-js-utils`](https://github.com/Laserfiche/lf-js-utils)
+`LfLocalizationService` is a service that handles internationalization and localization, mapping text to its corresponding translated texts in the selected language. `@laserfiche/resource-manager` provides language resources files in JSON that can be utlized by LfLocalizationService.
```
Comment (alexgomezlf) [inline README.md:18]: @laserfiche/lf-resource-library?

**Review by alexgomezlf — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (Ke-Jin-Laserfiche) [reply README.md]: good catch!

**Review by alexgomezlf — APPROVED** [review]

**Pull request merged**